experiment.py

```diff
@@ -93,24 +157,27 @@ def run_fig2_exp(schemes, args, run_full):
         downlink_trace = os.path.join(TRACE_DIR, downlink_ext)
     else:
         raise ValueError("Unknown experiment: %s" % exp)
+
+    if args.tiny_trace:
+        uplink_trace += "-tiny"
+        downlink_trace += "-tiny"
  
-    # Run each scheme experiment
-    
+    # Run experiment for each scheme 
     for scheme in schemes:
         print(" ---- Running Experiment %s for protocol: %s ---- \n" % (exp, scheme))
         protocol = get_protocol(scheme, uplink_ext, downlink_ext)
         cmds = protocol.get_figure2_cmds(delay, uplink_trace, downlink_trace, args)
         
         
         if scheme in run_full:
-            for c in cmds:
-                print("$ %s" % ' '.join(c.split(' ')))
-                call(c, shell=True)
+           run_cmds(cmds, args.verbose)
         else:
             print(" Experiment skipped ")
 
         print_fig2_results(protocol)
-        print(" ---- Done ---- \n")
+        time.sleep(2)
```

Comment:
Why sleep?

Reply:
Safety measure between experiments.  I figure that if some ports need to be cleared after the processes that used them have been killed, etc, giving some small time between experiments may be good practice